include/villas/config_helper.hpp

```diff
@@ -42,5 +46,102 @@ int json_object_extend(json_t *orig, json_t *merge);
 
 json_t *json_load_cli(int argc, const char *argv[]);
 
+template <typename Duration = std::chrono::nanoseconds>
+Duration parse_duration(const std::string &input) {
```

Comment:
Don't use `std::string const &` here. Prefer `std::string_view` for non-owning reference to string-like objects.